core/src/main/java/io/th0rgal/oraxen/OraxenPlugin.java

```diff
@@ -119,6 +121,10 @@ public void onEnable() {
         RecipesManager.load(this);
         invManager = new InvManager();
         ArmorEquipEvent.registerListener(this);
+        if (Settings.REIMPLEMENT_NOTEBLOCK_FEATURES.toBool()) {
+            Bukkit.getPluginManager().registerEvents(new TuneBlockListeners(), this);
```

Comment:
this should be registered in NoteBlockMechanicFactory and handled with that.
Otherwise this will run even when said mechanic is disabled